src/v2/Apps/Conversation/Components/Message.tsx

```diff
@@ -39,6 +40,15 @@ const AttachmentContainer = styled(Flex)<
   justify-content: space-between;
 `
 
+const MessageText = styled(Sans)`
+  white-space: pre-line;
```

Comment:
Both [CMS](https://github.com/artsy/volt/blob/107194e402f9ad73269c4888a6629dbedaf43d2b/app/helpers/conversations_helper.rb#L85) and [Radiation](https://github.com/artsy/radiation/blob/2e780f978a18ed9959c5863cdb10dbfd088074a0/app/views/message_mailer/delivery.html.mjml#L39) uses Rails [`simple_format`](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-simple_format) helper to convert the message body. Let's start from the simple CSS approach and can iterate later if we want to style messages in a more defined way (e.g. one newline char maps to `<br />` and 2 newline chars wraps text in a `<p>`).